tiledb/sm/c_api/tiledb.cc

```diff
@@ -3272,18 +3272,97 @@ int32_t tiledb_array_get_non_empty_domain(
   return TILEDB_OK;
 }
 
+int32_t tiledb_array_get_non_empty_domain_from_index(
+    tiledb_ctx_t* ctx,
+    tiledb_array_t* array,
+    unsigned idx,
```

Comment:
should we use `uint32_t` here instead of `unsigned`?

Reply:
Yeap, will fix.